vignettes/articles/05-read-the-outputs.Rmd

```diff
@@ -280,54 +334,61 @@ or bank for which the analysis was run.
+the following set of input parameters:
+
+* `asset_type_arg` is the type of the analysed financial asset in the portfolio. 
+* `lgd_senior_claims_arg` is the loss given default used in the expected loss
+calculation of loans for senior claims (notably for corporate loans, so this is
+a potentially important parameter in this calculation)
+* `lgd_subordinated_claims_arg` is the loss given default used in the expected loss
+calculation of loans for subordinated claims (notably for corporate bonds. Not
+of relevance in the loans calculation)
+* `risk_free_rate_arg` the risk free interest rate. Input parameter to the Merton
+credit risk model.
+* `discount_rate_arg` the annual rate at which future cash flows are discounted when
+calculating the net present value of companies in the loan book. Since the net
+present values are indirect inputs into the Merton credit risk model, this
+parameter my be relevant.
+* `div_netprofit_prop_coef_arg` Coefficient that indicates what share of the net
+present value should be considered to derive the equity value of a company.
+* `shock_year_arg` year in which the policy is introduced, that adjusts the
+production values of companies to stay in line with carbon budgets.
+* `fallback_term_arg` indicates which maturity the loans in the loan book are
+assumed to have. At this point we assume these are rolled over every year.
+* `scenario_geography_arg` Indicates which geographical region(s) (concerning asset
+location) results shall be calculated for.
+* `company_exclusion_arg` Logical parameter that indicates if certain combinations
+of company and technology should be excluded from the analysis. At this point,
+it is strictly recommended to set this to TRUE (default) because there is not
+yet a dedicated mechanism in place that handles companies which are phasing out
+certain technologies within the time frame of the production forecast.
+* `use_company_terms_arg` Logical parameter, indicating if individually provided
+terms for companies as configured in company_terms.csv shall be used instead of
+fallback as provided in `fallback_term`. The default is FALSE. For more details
+compare [Advanced data
+settings](https://2degreesinvesting.github.io/r2dii.climate.stress.test/articles/articles/04-advanced-data-settings.html).
+
+While there may be some redundancy to this information, it is there to ensure 
+the results remain reproducible and straight forward for the user
+to understand given magnitudes in the model results. In the standard run of the 
+stress test, with single value set for each input parameter, this implies that the 
+argument columns just contain one value each (as opposed to sensitivity analysis
+where the user can set multiples values of input parameters).
+
+### Sensitivity Analyses
+
+Running a sensitivity analysis produces as many sets of results as they are input
```

Comment:
"as many sets of results as the number of input parameters selected for the variable of interest."?